### ISSUE TITLE
Back In Time - Backups Failing

### ISSUE THREAD

**Issue by rvnicholas** [bit-team/backintime#889]
I have inherited responsibility for managing the Backup of our Ubuntu based, office file server which uses BIT to backup files from an NTFS disk (samba share) to an external USB drive (also ntfs).

I am relatively new to linux and haven't needed to look at this process until I was advised that one of the backup drives had died. 

Since replacing that drive and attempting to resurrect the process the backup appears to be failing:

- BIT "timeline" only shows "NOW" (I am expecting to see a date / time of a completed snapshot?)

- The status bar shows: "Working: <filename>" but isn't progressing and appears to be hanging

- The NOW snapshot log shows many occurences of "rsync: failed to set times on <filename> : Operation not permitted (1). I note however that some of the files named here ARE appearing in the backup location

So in summary, there is a lot of data making it to the destination drive but the process doesn't appear to be running to completion and is generating errors.

Unfortunately (as I say) I'm relatively new to Linux so and very much fumbling my way through this process - I'd appreciate any help although I may need help navigating around etc.

Many thanks in advance for your time.

**Comment by plgruener** [comment]
Hi, this could be
a) a problem with mount permissions. You should compare the mount-options for the old and the new drive or
b) BIT running as a wrong user (eg. as normal user, but trying to backup system-data in / without root permissions)
c) I once had the problem that it tried to backup the lost+found, which it naturally could not, resulting in similar error messages. Simply excluding lost+found solved that.

**Comment by buhtzz** [comment]
@rvnicholas There is a new forming maintaining team and we do review all issues. Is this problem still relevant for you or did you find a solution?

Tag: Feedback

**Comment by emtiu** [comment]
Closing this Issue because it's several years old, and we cannot reproduce the problem.
